src/chrome/content/rules/Etiqa.com.my.xml

```diff
@@ -0,0 +1,51 @@
+<!--
+	For other Maybank coverage, see Maybank.com.xml
+
+
+	Non-functional subdomains:
+
+		- etiqa.com.my	(h)
```

Comment:
I got different content (404) on https

Reply:
redirects to http for me including `www`

Reply:
See `http://etiqa.com.my/en/personal` and `https://etiqa.com.my/en/personal` (same for `www`)